docs/features/network-security-controls/admin-network-policy.md

```diff
@@ -1,303 +1,1118 @@
-    action: "Deny"
+  - name: "allow-from-ingress-nginx" # rule0
+    action: "Allow"
     from:
     - namespaces:
-        namespaceSelector:
-          matchLabels:
-            conformance-house: slytherin
-  egress:
-  - name: "deny-all-egress-to-slytherin"
+        matchLabels:
+          kubernetes.io/metadata.name: ingress-nginx
+  - name: "allow-from-monitoring" # rule1
+    action: "Allow"
+    from:
+    - namespaces:
+        matchLabels:
+          kubernetes.io/metadata.name: monitoring
+    ports:
+    - portNumber:
+        protocol: TCP
+        port: 7564
+    - namedPort: "scrape"
+  - name: "allow-from-open-tenants" # rule2
+    action: "Allow"
+    from:
+    - namespaces: # open tenants
+        matchLabels:
+          tenant: open
+  - name: "pass-from-restricted-tenants" # rule3
+    action: "Pass"
+    from:
+    - namespaces: # restricted tenants
+        matchLabels:
+          tenant: restricted
+  - name: "default-deny" # rule4
     action: "Deny"
+    from:
+    - namespaces: {}
+  egress:
+  - name: "allow-to-dns" # rule0
+    action: "Allow"
     to:
-    - namespaces:
+    - pods:
         namespaceSelector:
-          matchLabels:
-            conformance-house: slytherin
+          matchlabels:
+            kubernetes.io/metadata.name: kube-system
+        podSelector:
+          matchlabels:
+            app: dns
+    ports:
+    - portNumber:
+        protocol: UDP
+        port: 5353
+  - name: "allow-to-kapi-server" # rule1
+    action: "Allow"
+    to:
+    - nodes:
+        matchExpressions:
+        - key: node-role.kubernetes.io/control-plane
+          operator: Exists
+    ports:
+    - portNumber:
+        protocol: TCP
+        port: 6443
+  - name: "allow-to-splunk" # rule2
+    action: "Allow"
+    to:
+    - namespaces:
+        matchlabels:
+          tenant: splunk
+    ports:
+    - portNumber:
+        protocol: TCP
+        port: 8991
+    - portNumber:
+        protocol: TCP
+        port: 8992
+  - name: "allow-to-open-tenants-and-intranet-and-worker-nodes" # rule3
+    action: "Allow"
+    to:
+    - nodes: # worker-nodes
+        matchExpressions:
+        - key: node-role.kubernetes.io/worker
+          operator: Exists
+    - networks: # intranet
+      - 172.30.0.0/30
+      - 10.0.54.0/19
+      - 10.0.56.38/32
+      - 10.0.69.0/24
+    - namespaces: # open tenants
+        matchLabels:
+          tenant: open
+  - name: "pass-to-restricted-tenants" # rule4
+    action: "Pass"
+    to:
+    - namespaces: # restricted tenants
+        matchLabels:
+          tenant: restricted
+  - name: "default-deny"
+    action: "Deny"
+    to:
+    - networks:
+      - 0.0.0.0/0
 ```
 
-**OVN-Implementation:**
+* `subject` is the set of pods selected by `cluster-control` ANP, on
+  which the rules in the policy are applied on. Here its all the pods
+  in the cluster selected by the empty namespace selector.
+* Each AdminNetworkPolicy CRD will have a`.spec.priority` field. `priority`
```

Comment:
```suggestion
* Each AdminNetworkPolicy resource will have a`.spec.priority` field. `priority`
```

A CRD is a Custom Resource _Definition_. The ANPs themselves are "resources" (or "custom resources" but there's usually no reason to clarify that they're "custom").

Reply:
yeah thanks dan fixed it.